### DISCUSSION TITLE
v0.8.0-beta.18

### DISCUSSION THREAD

**Discussion by WebFreak001** [Pure-D/serve-d#376] [Announcements]
## What's Changed
* add Zed editor guide by @gdamore in https://github.com/Pure-D/serve-d/pull/372
* Adds support for dscanner.excludedFiles by @Mai-Lapyst in https://github.com/Pure-D/serve-d/pull/373

## New Contributors
* @Mai-Lapyst made their first contribution in https://github.com/Pure-D/serve-d/pull/373

**Full Changelog**: https://github.com/Pure-D/serve-d/compare/v0.8.0-beta.17...v0.8.0-beta.18

<hr /><em>This discussion was created from the release <a href='https://github.com/Pure-D/serve-d/releases/tag/v0.8.0-beta.18'>v0.8.0-beta.18</a>.</em>

**Comment by gsouf** [comment]
@WebFreak001  thanks for the work! I noticed that version 8 is still marked beta for 2 years but ships a very useful feature that is `go to references` which seems absent in 7.x

Any plans to make it a stable release?
